lib/shopify-cli/core/monorail.rb

```diff
@@ -99,7 +99,7 @@ def build_payload(start_time, end_time, args, err = nil)
 
               if Project.has_current?
                 project = Project.current
-                payload[:api_key] = project.env.api_key
+                payload[:api_key] = project.env.api_key unless Project.current_project_type.to_s == 'script'
```

Comment:
Perhaps we could add a method such as `current_project_is_script?` method on `Project`?

Reply:
The motivation behind this is that this line throws an exception in the case of a scripts project (it can't find a .env file), and this throws an exception on the post.body line below and the message isn't sent to monorail.

```
post.body = JSON.dump(build_payload(start_time, end_time, args, err))
http.request(post)
```

Reply:
What is the error? You probably need to change the data type to an empty string rather than a nil value instead of this special scripts case

Reply:
The error is `#<Errno::ENOENT: No such file or directory @ rb_sysopen - /Users/erik/src/github.com/Shopify/shopify-app-cli/eriktestsc/.env>`. Empty string sounds like a good idea.